Cargo.toml

```diff
@@ -1,2 +1,2 @@
 [workspace]
-members = ["crates/*", "xtask"]
+members = ["crates/*", "xtask", "examples/*"]
```

Comment:
Can we sort alphabetically?

```suggestion
members = ["crates/*", "examples/*", "xtask"]
```

Reply:
Cargo sort should do this, is this just failing because it's panicking before this maybe?

Reply:
Maybe that is related to the crash even!